scripts/post-build

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Any way to support windows <10 which do not have bash? Maybe have a node.js script and have the script in the package.json be `node <file>`

Reply:
I tried to emulate the other scripts that were there. Let me investigate a little further. If we do `node file` does it autochoose the file/extension based on OS? Example of that somewhere?

Reply:
Those scripts don't work for me (on windows), I have to run those steps manually (which is easy since they are mostly git commands) and they only run on the "release". This is in the build, which is probably more common for others to do.